src/main/java/org/jabref/gui/actions/ActionFactory.java

```diff
@@ -74,7 +74,7 @@ private static Label getAssociatedNode(MenuItem menuItem) {
     }
 
     public MenuItem configureMenuItem(Action action, Command command, MenuItem menuItem) {
-        ActionUtils.configureMenuItem(new JabRefAction(action, command, keyBindingRepository), menuItem);
+        ActionUtils.configureMenuItem(new JabRefAction(action, command, keyBindingRepository, "Menu"), menuItem);
```

Comment:
Could you please create a new enum for the `Menu` and `Button` constants (I don't like magic strings).